cypress/integration/basic-math-operations.spec.js

```diff
@@ -40,7 +40,7 @@ describe("basic math operations", () => {
   it("should show message error if divide by zero", () => {
     cy.get(".nine").click();
     cy.get(".division").click();
-    cy.get(".Zero").click();
+    cy.get(".zero").click();
```

Comment:
Good catch! 👍

Reply:
I'm wondering why Cypress didn't fail 🤔 
Because there is for sure no element with a class `.Zero` 😕